dev-python/elasticsearch-py/files/elasticsearch-py-5.5.1-urllib.patch

```diff
@@ -0,0 +1,22 @@
+From ea5d5fdebfae9b8a7a04d5951d87b3cb9c86246c Mon Sep 17 00:00:00 2001
+From: Nick Lang <nick@nicklang.com>
+Date: Wed, 15 Nov 2017 08:59:56 -0500
+Subject: [PATCH] fixing a version requirement for urllib3 on 5.x branch
+
+---
+ setup.py | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index c320461..2343d89 100644
+--- a/setup.py
++++ b/setup.py
+@@ -12,7 +12,7 @@
+ f.close()
+ 
+ install_requires = [
+-    'urllib3<1.22,>=1.21.1',
++    'urllib3<1.23,>=1.21.1',
```

Comment:
/me not get it. Wouldn't it be be better to patch or sed it out completely instead of adjusting one by one? Especially given that you can always have more strict deps in ebuild.

Reply:
This is an official patch from upstream so I thought it would be best to stay as vanilla as possible. Only upstream "guarantees" which urllib versions this will work with.

Reply:
Sure but it feels kinda silly to have a 'urllib' patch which turns out not to alter code at all, and have to update it later when it turns out 1.23 works as well. `sed -i -e /urllib3/d setup.py || die` would be lighter and more future-proof. If 1.23 doesn't work, you can keep <-dep in ebuild.

Reply:
Ok, I'll update the PR in that regard.